### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by Farida-Hamid**
>In this project, I implemented the CRUD (create, update, delete) methods. All the elements of the user interface are now fully functional.

### Steps:
- Remove all hardcoded items from the tasks array.
- Implement a function for adding a new task.
- Implement a function for deleting a task.
- Implement a function for editing task descriptions.
- By default new tasks have the property completed set to false and the property index set to the value of the new array length.
- Deleting a task updates all remaining items' indexes.
- All changes to the To-Do List are saved in local storage.

### CODE REVIEW

File: src/tasks-manager.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  add = (value) => {
+    const task = {
+      description: value,
+      completed: false,
+      index: this.list.length,
+    };
+    this.list.push(task);
+    localStorage.setItem('memory', JSON.stringify(this.list));
+    this.loader();
+  }
```
Comment (AmaduKamara) [inline src/tasks-manager.js:66]: - [x] The addition of todo functionality does not seem to work as you implemented it. The inputted data is not being submitted when the enter key is pressed. Kindly work on that and be sure a todo is added and stored to localStorage and then todos are fetched from localStorage then rendered to the screen. Please work on that 👍 

- [x] Currently, adding a todo is not possible and hence the other functionalities cannot be tested, editing or updating a todo and deleting a todo. Kindly work on all those features and be sure they are all functioning well and then ask for a review 👍 

image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️

Hi @Farida-Hamid,

WeHaveToTryAirspeederGIF (2)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct use of Github workflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly make it possible for a user to be able to add a todo
- [x] A todo is updated
- [x] A todo can be deleted. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/tasks-manager.js (hunk 1 above)
Comment (Farida-Hamid) [reply src/tasks-manager.js:66]: All done thank you

**Review by phelian23 — APPROVED** [review]
Hi @Farida-Hamid,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**